From d52f850bd2d1c07316669ebc05690917cc814237 Mon Sep 17 00:00:00 2001
From: Purrr <masamik2222@gmail.com>
Date: Mon, 4 Nov 2024 13:57:01 +0700
Subject: [PATCH] lib: zstd: change fallthrough definitions

As instructed in the comments, our kernel does not support this feature due to its outdated version.
---
 lib/zstd/common/compiler.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/zstd/common/compiler.h b/lib/zstd/common/compiler.h
index c437e0975575..51c09d64756a 100644
--- a/lib/zstd/common/compiler.h
+++ b/lib/zstd/common/compiler.h
@@ -156,7 +156,7 @@
  * - CPP17: https://en.cppreference.com/w/cpp/language/attributes/fallthrough
  * - Else: __attribute__((__fallthrough__))
  */
-#define ZSTD_FALLTHROUGH fallthrough
+#define ZSTD_FALLTHROUGH __attribute__((__fallthrough__))
 
 /*-**************************************************************
 *  Alignment check
